public/index.html

```diff
@@ -142,7 +147,11 @@
             }
           },
           history: annotationHistory,
-        });
+        };
+
+        Object.assign(lsfConfig, window.LSF_CONFIG ?? {});
```

Comment:
I think you can just use `assignConfig()` method

Reply:
turned out we don't need it at all